docs/reference/remap/expressions/if.cue

```diff
@@ -40,6 +40,7 @@ remap: expressions: if: {
 			source: #"""
 				if false {
 					# not evaluated
+					null
```

Comment:
Empty blocks aren't allowed, and whitespace is (currently) stripped from the AST so doesn't count.